setup.py

```diff
@@ -51,7 +51,7 @@
     package_dir={'': 'src'},
     packages=find_packages('src'),
     package_data={
-        '': ['data/terraform/*']
```

Comment:
any reason not including `https://github.com/adobe/ops-cli/tree/master/src/ops/data/terraform` as well?

Reply:
We removed that part since it doesn't belong to kompo's configuration. You can configure terraform separately.